File: u-admin/src/main/java/com/study/u/dataobject/Product.java
package com.study.u.dataobject;

import lombok.Data;

import javax.persistence.Entity;
import javax.persistence.Id;
import javax.persistence.Table;

@Data
@Entity
@Table(name = "u_product")
public class Product {

    @Id
    private Integer id;

    /** 期限 单位天. */
    private Integer time;

    /** 期限中文描述. */
    private String time_desc;

    /** 收益百分比. */
    private Integer gain;

    /** 条件（U币个数）. */
    private Integer gainCondition;

}

File: u-android/app/src/main/java/com/study/u/data/Product.kt
package com.study.u.data

import com.squareup.moshi.JsonClass

@JsonClass(generateAdapter = true)
data class Product(
    var id: Int,
    /** 期限 单位天. */
    var time: Int,
    /** 期限中文描述.  */
    var time_desc: String,
    /** 收益.  */
    var gain: Int,
    /** 条件.  */
    var gainCondition: Int
)